internal/pkg/manifest/env.go

```diff
@@ -247,17 +251,20 @@ func (cfg *environmentVPCConfig) ManagedVPC() *template.ManagedVPC {
 	// NOTE: sort based on `az`s to preserve the mappings between azs and public subnets, private subnets.
 	// For example, if we have two subnets defined: public-subnet-1 ~ us-east-1a, and private-subnet-1 ~ us-east-1a.
 	// We want to make sure that public-subnet-1, us-east-1a and private-subnet-1 are all at index 0 of in perspective lists.
-	sort.Slice(cfg.Subnets.Public, func(i, j int) bool {
+	sort.SliceStable(cfg.Subnets.Public, func(i, j int) bool {
 		return aws.StringValue(cfg.Subnets.Public[i].AZ) < aws.StringValue(cfg.Subnets.Public[j].AZ)
 	})
-	sort.Slice(cfg.Subnets.Private, func(i, j int) bool {
+	sort.SliceStable(cfg.Subnets.Private, func(i, j int) bool {
 		return aws.StringValue(cfg.Subnets.Private[i].AZ) < aws.StringValue(cfg.Subnets.Private[j].AZ)
 	})
 	for idx, subnet := range cfg.Subnets.Public {
 		publicSubnetCIDRs[idx] = aws.StringValue((*string)(subnet.CIDR))
 		privateSubnetCIDRs[idx] = aws.StringValue((*string)(cfg.Subnets.Private[idx].CIDR))
 		azs[idx] = aws.StringValue(subnet.AZ)
 	}
+	if azs[0] == "" {
+		azs = nil
+	}
```

Comment:
this looks so weird...can we just make `var azs []string`?

Reply:
updated!